tests/src/autoload/functions.php

```diff
@@ -776,11 +776,12 @@ function loadDataset()
 
 /**
  * Test helper, search an item from its type and name
- *
- * @param string    $type
- * @param string    $name
- * @param bool      $onlyid
- * @return CommonDBTM|int the item, or its id
+ * @template T of CommonDBTM
+ * @param class-string<T>   $type
+ * @param string            $name
+ * @param bool              $onlyid
+ * @phpstan-return ($onlyid is true ? Int : T)
```

Comment:
There was probably a mistake here, it should be `int` (vscode doens't like it).

![image](https://github.com/user-attachments/assets/1ed38913-2bfe-401a-87e5-acd06880a608)

Reply:
fixed in 99ae6637566de57538cb77b687419175126464e9